packages/backend/src/api/factory.ts

```diff
@@ -47,6 +50,27 @@ export function createBackendApiClient(options: CreateBackendApiOptions) {
     emailAddresses: new EmailAddressAPI(request),
     instance: new InstanceAPI(request),
     invitations: new InvitationAPI(request),
+    // Using "/" instead of an actual version since they're bapi-proxy endpoints.
+    // bapi-proxy connects directly to C1 without URL versioning,
+    // while API versioning is handled through the Clerk-API-Version header.
+    machineTokens: new MachineTokensApi(
+      buildRequest({
+        ...options,
+        apiVersion: '/',
+      }),
+    ),
+    idPOAuthAccessToken: new IdPOAuthAccessTokenApi(
+      buildRequest({
+        ...options,
+        apiVersion: '/',
+      }),
+    ),
+    apiKeys: new APIKeysAPI(
+      buildRequest({
+        ...options,
+        apiVersion: '/',
+      }),
+    ),
```

Comment:
These are just placeholder versions. When normalized, the "/" will be removed from the URL path while still maintaining version control through the `Clerk-API-Version` header. bapi-proxy does not have it in the URL